app/content/react/library-reference/widgets.md

```diff
@@ -26,8 +26,9 @@ Renders a `<CategoryWidget />` component, binded to a source at redux. The widge
 | props.dataSource        | <code>string</code>   |                 | ID of the data source to get the data from.                                                                                         |
 | props.column            | <code>string</code>   |                 | Name of the data source's column to get the data from.                                                                              |
 | props.operation         | <code>string</code>   |                 | Operation to apply to the operationColumn. Must be one of those defined in `AggregationTypes` object.                               |
-| [props.operationColumn] | <code>string</code>   |                 | (optional) Name of the data source's column to operate with. If not defined, same as `column`.                                      |
-| [props.animation]       | `bool`                | `true`          | (optional) Indicates whether the widget update is animated or jumps directly to the new state. |
+| [props.operationColumn] | <code>string</code>   |                 | (optional) Name of the data source's column to operate with. If not defined, same as `column`. If multiples are provided, they will be merged into a single one using joinOperation property. |
```

Comment:
I'd change the description to:

(optional) Name of the data source's column to operate with. If not defined, same as `column`. If multiple columns are provided, they will be combined using the operation specified with the `joinOperation` property.